src/plugins/add-cal/reports/a11y-1-en.hbs

```diff
@@ -0,0 +1,75 @@
+---
+{
+	"title": "Add to Calendar",
```

Comment:
```suggestion
	"title": "Accessibility assessment #1 - Add to Calendar",
```

Reply:
Commenting to resolve conversation, changes were applied in previous commit